dependencies.yaml

```diff
@@ -279,7 +279,7 @@ dependencies:
       - output_types: [requirements, pyproject]
         packages:
           # Allow runtime version to float up to minor version
-          - pyarrow==12.*
+          - pyarrow==13.*
```

Comment:
Merge this with the `conda` dependency list above. Look at the previous PR to see how this was split up, and merge it back together by reverting those changes to the structure.

Reply:
Done.